azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -626,6 +622,14 @@ private void doRun() {
     writeStatus();
   }
 
+  private void uploadExecutableNode() {
+    try {
+      this.loader.uploadExecutableNode(this.node, this.props);
+    } catch (final ExecutorManagerException e1) {
```

Comment:
Thanks for catching this issue!
Can we also refactor this method a bit?
e.x: final ExecutorManagerException e instead of e1
and log the exception message when catch the exception.